format_spec/array_file_hierarchy.md

```diff
@@ -3,16 +3,19 @@
 An array is a folder with the following structure:
 
 ```
-my_array                          # array folder
-    |_ __schema                   # array schema folder
-    |_ <timestamped_name>         # fragment folder
-    |_ <timestamped_name>.ok      # fragment ok file
-    |_ ...
-    |_ <timestamped_name>.vac     # fragment vacuum file
-    |_ ...                  
-    |_ <timestamped_name>.meta    # consol. fragment meta file
-    |_ ...                  
-    |_ __meta                     # array metadata folder
+my_array                                # array folder
+    |_ __schema                         # array schema folder
+    |_ __fragments                      # array fragments folder
+          |_ <timestamped_name>         # fragment folder
+          |_ ...
+    |_ commits                          # array commits folder
+          |_ <timestamped_name>.ok      # fragment ok file
```

Comment:
I suggest we rename `*.ok` to `*.wrt` (for writes). We will have deletion and update support very soon, which will generate files `*.del` and `*.upd` respectively. In that case `*.ok` will stop being self-explanatory any more.

Reply:
I see that `*.wrt` is mentioned below, so this is probably a typo here.